sites/website/src/docs/design-systems/high-contrast.md

```diff
@@ -0,0 +1,184 @@
+        background: ${accentFillActiveBehavior.var};
+    }
+    :host([appearance="accent"]) .control:${focusVisible} {
+        box-shadow: 0 0 0 calc(var(--focus-outline-width) * 1px) inset ${neutralFocusInnerAccentBehavior.var};
+    }
+    :host([appearance="accent"][disabled]) {
+        opacity: var(--disabled-opacity);
+        background: ${accentFillRestBehavior.var};
+    }
+`
+```
+
+When high contrast is enabled, the system will try to apply the correct color. In the case of this accent button, the system is missing a few things. We do not have a background, `rest` and `hover` states are the same, `focus` is not following the button's `focus` design, and the `disabled` state is too dim.
+
+![Accent button no forced colors](https://static.fast.design/assets/high-contrast/accent-no-forced-colors.png)
+
+To fix this, we will pass a [forcedColorsStylesheetBehavior](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/utilities/match-media-stylesheet-behavior.ts) object to `withBehaviors`, using similar selectors, and setting property values with the `SystemColors` keyword.
+
+```ts
+export const AccentButtonStyles = css`
+    /* ... */
+`.withBehaviors(
+    forcedColorsStylesheetBehavior(
+        css`
+            :host([appearance="accent"]) .control {
+                forced-color-adjust: none;
+                background: ${SystemColors.Highlight};
+                color: ${SystemColors.HighlightText};
+            }
+            :host([appearance="accent"]) .control:hover,
+            :host([appearance="accent"]:active) .control:active {
+                background: ${SystemColors.HighlightText};
+                border-color: ${SystemColors.Highlight};
+                color: ${SystemColors.Highlight};
+            }
+            :host([appearance="accent"]) .control:${focusVisible} {
+                border-color: ${SystemColors.ButtonText};
+                box-shadow: 0 0 0 2px ${SystemColors.HighlightText} inset;
+            }
+            :host([appearance="accent"][disabled]),
+            :host([appearance="accent"][disabled]) .control,
+            :host([appearance="accent"][disabled]) .control:hover {
+                background: ${SystemColors.ButtonFace};
+                border-color: ${SystemColors.GrayText};
+                color: ${SystemColors.GrayText};
+                opacity: 1;
+            }
+        `
+    )
+);
+```
+
+After adding `forced-colors` and applying `SystemColors` keywords, the accent button now uses `Highlight` as a background for its `rest` state. On the `hover` and `active` states, the background and color from the `rest` state are swapped. A double border treatment is applied when in the `focus` state, and the `disabled` has opacity set to 1 and uses the disabled color, `GrayText`, for color on the border and content.
+
+![Accent button forced colors](https://static.fast.design/assets/high-contrast/accent-with-forced-colors.png)
+
+:::note
```

Comment:
Is the triple colon note a standard that we use? I don't see it in Markdown documentation and it isn't formatting this block... is it supposed to?

Reply:
@brookdozer that is a good question. I saw that our "FAST Frame" and "Creating a Design System" documentation was using it, so I did the same.